tests/test_func.py

```diff
@@ -201,3 +204,52 @@ def my_sum(val_list: List[int]) -> int:
     assert len(results) == 2
     for row in results:
         assert ("is_even" in row) and (row["is_even"] is not None) and (row["result"] == 5)
+
+
+def test_func_return_comp_type(db: gp.Database):
+    class Person:
+        _first_name: str
+        _last_name: str
+
+    @gp.create_function
+    def create_person(first: str, last: str) -> Person:
+        return {"first_name": first, "last_name": last}
+
+    for row in create_person("'Amy'", "'An'", db=db).to_table().fetch():
+        assert row["first_name"] == "Amy" and row["last_name"] == "An"
+
+
+def test_func_comp_type_column(db: gp.Database):
```

Comment:
This test is not quite different from `test_func_return_comp_type`, what is the main purpose of this test? should it have a different name?
Is the original purpose of this test to test a composite-type column? but the column type is `int` I think?